dashboard/db/migrate/20180702201106_add_deleted_at_to_teacher_feedbacks.rb

```diff
@@ -0,0 +1,5 @@
+class AddDeletedAtToTeacherFeedbacks < ActiveRecord::Migration[5.0]
+  def change
+    add_column :teacher_feedbacks, :deleted_at, :datetime, default: nil
+  end
```

Comment:
Question: Do we have to provide `up` and `down` methods in these migrations, or is Rails smart enough to be able to reverse this change if needed?

Reply:
Rails is smart enough for certain types of migrations, `add_column` being one of them. I don't know a definitive list, but typically anything that has enough information to derive the reverse does not need an explicit `down` migration. In this case, reversing adding a column is deleting the column and all that's needed is the table and column name so this is sufficient.

In any case, we should always run the migrate and rollback to be sure. I always run these commands locally, and make sure they each succeed, before committing any migrations:

```
cd dashboard
rake db:migrate
rake db:rollback
rake db:migrate
```